auto_dev/commands/augment.py

```diff
@@ -427,33 +430,46 @@ def skill_from_fsm(spec_file: str, skill_public_id: PublicId, auto_confirm: bool
         auto_confirm (--auto-confirm): Auto confirm the augmentation
 
         verbose (--verbose): Verbose output
-
+        force (--force): Force the augmentation
     Usage:
 
         adev augment skill_from_fsm fsm_spec.yaml author/skill_name:0.1.0
 
     """
 
     if not Path(spec_file).exists():
-        logger.error(f"Specification file for FSM not found: {spec_file}")
-        sys.exit(1)
+        msg = f"Specification file for FSM not found: {spec_file}"
+        logger.error(msg)
+        raise OperationError(msg)
     if not Path(AEA_CONFIG).exists():
-        logger.error(f"File {AEA_CONFIG} not found")
-        sys.exit(1)
+        msg = f"File {AEA_CONFIG} not found"
+        logger.error(msg)
+        raise OperationError(msg)
     if not skill_public_id:
-        logger.error("Skill public id not provided. Unsure which skill to augment.")
+        msg = "Skill public id not provided. Unsure which skill to augment."
+        logger.error(msg)
+        raise OperationError(msg)
 
     skill_dir = Path(f"{SKILLS}/{skill_public_id.name}")
+    behaviour_path = skill_dir / "behaviours.py"
     config, *_overrides = load_autonolas_yaml(PackageType.SKILL, skill_dir)
 
     if config.get("author") != skill_public_id.author or config.get("name") != skill_public_id.name:
-        logger.error(f"Skill {skill_public_id} not found in the current project.")
-        sys.exit(1)
```

Comment:
Shoudl defintielt be exiting wiht 1 i think as this is cli command group, not the actual business logic